ee-9/feature-pack/pom.xml

```diff
@@ -1046,6 +1050,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>${ee.maven.groupId}</groupId>
+            <artifactId>jipijapa-eclipselink-jakarta</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Hi @boris-unckel , just in case you missed it on the wildfly google groups, notice that we have changed the structures of the wildfly preview FP and this dependency and its license file now should be placed in `ee-9/common` maven module.